src/configs/env.ts

```diff
@@ -359,3 +359,10 @@ export type JURISDICTION_METADATA_RISK = typeof JURISDICTION_METADATA_RISK;
 export const JURISDICTION_METADATA_RISK_PERCENTAGE =
   process.env.REACT_APP_JURISDICTION_METADATA_RISK_PERCENTAGE || '90';
 export type JURISDICTION_METADATA_RISK_PERCENTAGE = typeof JURISDICTION_METADATA_RISK_PERCENTAGE;
+
+/** plan types with map enabled during plan assignment */
+export const MAP_ENABLED_PLAN_TYPES = String(
+  process.env.REACT_APP_MAP_ENABLED_PLAN_TYPES ||
```

Comment:
I think that we should switch to using a blacklist i.e. disabled plan types.  This is so that when we add new plan types we don't have to remember to change this